internal/operators/cnv/manifest.go

```diff
@@ -6,26 +6,65 @@ import (
 )
 
 // Manifests returns manifests needed to deploy CNV
-func Manifests() (map[string][]byte, error) {
-	cnvSubs, err := subscription()
+func Manifests(config Config) (map[string][]byte, error) {
+	cnvSubs, err := subscription(config)
```

Comment:
Add `Manifest` to the following variable names.
```suggestion
	cnvSubsManifest, err := subscription(config)
```